buildSrc/assets/themes/chuunibyou/rikka/dark/rika.dark.chrome.definition.json

```diff
@@ -0,0 +1,14 @@
+{
+  "id": "5412c41d-f76b-4488-85a7-1ae1a63bbfcc",
+  "overrides": {
+    "theme": {
+      "colors": {},
```

Comment:
On of the things with chrome is that the `omnibox` likes to choose which background suites the current font. 

For Rikka & Raphtalia, I had to change it to a font that is more usabile for a dark background, so that the search box isn't light even though the theme is dark.

See https://github.com/doki-theme/doki-theme-web/blob/bf93ce316678da8a18a78fef337539cf6880e108/buildSrc/assets/themes/chuunibyou/rikka/dark/rikka.dark.chrome.definition.json#L6

![image](https://user-images.githubusercontent.com/15972415/153518174-a6701563-a006-4801-9248-03b2cf1a3f53.png)